backend/src/pike_defaults.py

```diff
@@ -0,0 +1,12 @@
+import os
+import src.pike_util as pike_util
```

Comment:
let's rename this to utils (since it is in pike, pike is redundant) or pike_utils.